packages/kununu-theme-v3/scss/typography.scss

```diff
@@ -2,31 +2,31 @@
 @import './mixins.scss';
 
 .h1 {
-  @include font-props($dark, 22px, -.27px, null, bold, center);
+  @include font-props($dark, 21px, -.2px, 1.52, bold, center);
 }
 
 .h2 {
-  @include font-props($charcoal-grey, 21px, -.3px, null, bold);
+  @include font-props($charcoal-grey, 18px, -.1px, 1.33, bold);
 
   & .center {
-    @include font-props(null, 22px, -.31px, 28px, null, center);
+    @include font-props(null, null, null, null, null, center);
```

Comment:
I can't remember of anything using this style. It looks safe to remove it

Reply:
I am hesitant to remove it completely even though it looks like it is not being used. There is a very similar class for `h6` which I also left in.